docs/readmes/feg/debug_cli.md

```diff
@@ -7,8 +7,7 @@ hide_title: true
 # FeG CLI
 
 Some Go services on FeG include a cli to run commands against the
-3GPP core. The different clients source code can be seen at
-[here](https://github.com/magma/magma/tree/master/feg/gateway/tools)
+3GPP core. The different clients source code can be seen [here](https://github.com/magma/magma/tree/master/feg/gateway/tools).
```

Comment:
Could we change this to "can be seen in the [`feg/gateway/tools`](...) folder"?
Current rule is that we do not use `here` for links - see [`docs`](https://github.com/magma/magma/wiki/Contributing-Documentation#styling-and-writing-conventions).